charts/auth-layer-server/Chart.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: v2
+name: auth-layer-server
+appVersion: "19.1.0"
```

Comment:
yeah. It seems like we should just mirror the keycloak chart value for version and appVersion here?

Reply:
yes, this is right and on purpose, since appVersion normally refers to the version of the app, in this case is version 19.1.0 of the Bitnami Chart, however I also see your point and either way it would work and be fine.